packages/@okta/vuepress-site/docs/reference/rest/index.md

```diff
@@ -149,6 +148,8 @@ First, create a service app integration where you can define your scope-based ac
    {style="list-style-type:lower-alpha"}
    1. Select **Super Administrator** under Roles and click **Save Changes**.
 
```

Comment:
Same comment about this setting being a temporary use case for customers.

Reply:
@vasishtajayanti-okta: Ok, I will remove the note for the Testing in Postman instructions (since it doesn't involve an existing script). However, I'll leave the note in the OAuth Org2Org guide, since there could be existing scripts for this flow (and currently, this org setting isn't clearly documented anywhere for users). Will this org setting be removed in the future? Please be aware that this affects many developer use cases, so contact Infodev for any changes. Thank you!  cc: @ramyamudduluri-okta